recipes/jmol/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "Jmol" %}
+{% set version = "14.29" %}
+{% set patchlevel = "15" %}
+{% set sha256 = "e71df634776ade7c57f58986affd2b68e0d0d61bcda7b031e0b51da9d4aa81e1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/jmol/files/Jmol/Version {{ version }}/Jmol {{ version }}.{{ patchlevel }}/Jmol-{{ version }}.{{ patchlevel }}-binary.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
Is there another `noarch:` value? This isn't Python of course…

Reply:
generic